include/SSVOpenHexagon/Core/HexagonDialogBox.hpp

```diff
@@ -17,31 +17,67 @@ namespace hg
 class HGAssets;
 class StyleData;
 
+enum DBoxDraw
+{
+    topLeft = 0,
+    center,
+    centerUpperHalf
+};
+
 class HexagonDialogBox
 {
 private:
+    using KKey = sf::Keyboard::Key;
+    using Color = sf::Color;
+    using Vector2 = sf::Vector2f;
```

Comment:
Mostly personal style but I've grown to avoid aliases whose only purpose is to make code shorter. Up to you, not asking you to remove them.